### ISSUE TITLE
Each as PRON instead of DET

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-LinES#11]
The standard in English treebanks is to label `each` as a DET:

```
# sent_id = weblog-blogspot.com_alaindewitt_20040929103700_ENG_20040929_103700-0079
# text = His column appears in The Hill each week.
1       His     his     PRON    PRP$    Case=Gen|Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs 2       nmod:poss       2:nmod:poss     _
2       column  column  NOUN    NN      Number=Sing     3       nsubj   3:nsubj _
3       appears appear  VERB    VBZ     Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin   0       root    0:root  _
4       in      in      ADP     IN      _       6       case    6:case  _
5       The     the     DET     DT      Definite=Def|PronType=Art       6       det     6:det   _
6       Hill    Hill    PROPN   NNP     Number=Sing     3       obl     3:obl:in        _
7       each    each    DET     DT      _       8       det     8:det   _
8       week    week    NOUN    NN      Number=Sing     3       obl:tmod        3:obl:tmod      SpaceAfter=No
9       .       .       PUNCT   .       _       3       punct   3:punct _
```

Any objections to updating the `each` UPOS from `PRON` to `DET`?

**Comment by rueter** [comment]
I trust this is on a by-context basis. I can also think of an ADV reading:

*They gave us two apples each.*

**Comment by AngledLuffa** [comment]
I will do it as a PR before making any final changes

**Comment by nschneid** [comment]
I think EWT should have a consistent policy on "each". (Demonstratives not in det function become PRON, but most other DETs stay DET: https://universaldependencies.org/en/pos/DET.html)

**Comment by AngledLuffa** [comment]
@nschneid that's not what happens in practice in EWT.  `each other` gets tagged `DET` with the feature `ExtPos=PRON`, and all of the others just get tagged `DET`.  For example:

```
# the spaces are each are actually tabs
grep " each    " extern_data/ud2/git/UD_English-EWT/en_ewt-ud-train.conllu | cut -f 4 | sort | uniq
DET
```

**Comment by nschneid** [comment]
Right, I'm saying only demonstratives and relativizers become PRON in EWT. "each" is neither so it stays DET.

**Comment by AngledLuffa** [comment]
Got it.  In that case, unless there are objections from the stakeholders,
I'll set each each to DET

**Comment by LarsAhrenberg** [comment]
> In that case, unless there are objections from the stakeholders,

I do object to anyone making changes to UD_English-LinES without my consent. I welcome that errors are pointed out, where they exist, but I much prefer to correct them myself. So, please don't.

While I appreciate the efforts to improve consistency between treebanks I fear it sometimes is going too fast. Many decisions are now being made relating to EWT and GUM that revoke earlier decisions and where's the guarantee that they will not be changed in the future? And minor inconsistencies cannot be that harmful. Moreover, increasing consistency in one place may cause inconsistency in others, e.g. between languages. I have annotated data that I maintain to add to the treebank in the future and that will obviously be less consistent with the public UD data if anyone can just go in and make changes. Besides, I think the decision to declare _each_ as always DET is misguided and one that may well be changed in the future. See the example above from @rueter.

**Comment by AngledLuffa** [comment]
That's fair, and it's not my intention to step on toes.  If you want the
"each" change unrolled, I can certainly do that.   I do think the intention
is to have UPOS be truly universal, though, and the expectation would be
for the same sentence to have the same tags across treebanks.  If there are
disagreements between treebank annotation schemes, it would be good to come
to a consensus.

**Comment by LarsAhrenberg** [comment]
To be consistent with myself I'll do a review of _each_ before Nov. 1st and change back, where necessary.

**Comment by AngledLuffa** [comment]
@LarsAhrenberg were you able to review my `each` change to match your standards?

**Comment by AngledLuffa** [comment]
Following up on this a while later - is this issue resolved?

**Comment by LarsAhrenberg** [comment]
I have updated the reciprocal fixed expression _each other_. As for substantive and adverbial uses it is a more general issue, as it would apply also to other words such as _some_, _any_, and more (as specified on the English page for DET:
 [https://universaldependencies.org/en/pos/DET.html]. I will explain my hesitation in a reply to your other issue regarding _after all_  [https://github.com/UniversalDependencies/UD_English-LinES/issues/19]
